### PR TITLE
PHPMailer Implementation

### PR DESCRIPTION
**Pull request by anthonyvscode**
Implemented [PHPMailer](https://github.com/PHPMailer/PHPMailer) as an alternative to using the `mail()` function so that SMTP credentials can be set. Note: `composer install` will need to be run on release process to get the `vendor/` folder in the project with PHPMailer

Should resolve issue https://github.com/geoffhumphrey/brewcompetitiononlineentry/issues/955

### CODE REVIEW

**Comment by ercousin** [comment]
This sounds awesome! @geoffhumphrey what do you think?

File: vendor/autoload.php [hunk 1]
```diff
@@ -0,0 +1,7 @@
+<?php
```
Comment (parkertr) [inline vendor/autoload.php]: the vendor directory should be added to the .gitignore file and not committed to this repo.

**Comment by anthonyvscode** [comment]
Btw - I've used these edits in two competition installs since I opened this PR without any issues 👍

**Comment by geoffhumphrey** [comment]
Thanks for your work on this @anthonyvscode. I'm cloning your fork now and will begin testing in the next few day. From the looks of things after reviewing the code, it's pretty a straight-forward integration and will definitely be included in the next release.

**Pull request merged**